app/views/films/index.html.erb

```diff
@@ -89,12 +93,56 @@
         <td><%= movie['release_date'] %></td>
         <td><%= movie['vote_average'] %></td>
         <td><%= movie['genres'] %></td>
-        <td style="max-width: 50px;"><%= image_tag movie['poster_path'], style: "max-width: 100%; height: auto;" %></td>
+        <td><%= image_tag movie['poster_path'], style: "max-width: 30px; height: auto;" %></td>
+      </tr>
+    <% end %>
+  <% end %>
+  </tbody>
+</table>
+<hr>
+</div>
+
+<hr>
+<caption><h1>New Movies From Api except Database</h1></caption>
+<div class="table-container">
+<table class="table table-bordered">
+  <thead>
+  <tr>
+    <th scope="col">No</th>
+    <th scope="col">Title</th>
+    <th scope="col">Release Date</th>
+    <th scope="col">Vote Average</th>
+    <th scope="col">Genres</th>
+    <th scope="col">Image</th>
+    <th scope="col">Actions</th>
+  </tr>
+  </thead>
+  <tbody>
+  <% @new_movies.in_groups_of(5, false) do |row| %>
+    <% row.compact.each_with_index do |movie, index| %>
+      <tr>
+        <th scope="row"><%= index + 1 %></th>
+        <td><%= movie['title'] %></td>
+        <td><%= movie['release_date'] %></td>
+        <td><%= movie['vote_average'] %></td>
+        <td><%= movie['genres'] %></td>
+        <td><%= image_tag movie['poster_path'], style: 'max-width: 30px; height: auto;' %></td>
+        <td>
+          <%= form_with(url: add_movie_path(movie_id: movie['_id']), method: :post) do |f| %>
+            <%= f.hidden_field :title, value: movie['title'] %>
+            <%= f.hidden_field :rating, value: movie['vote_average'] %>
+            <%= f.hidden_field :genre, value: movie['genres'] %>
```

Comment:
avoid doing this approach instead of this pass the whole object and use that at the backend.